tests/test_integration.py

```diff
@@ -33,3 +37,44 @@ def test_basic_integration(python_version, project_no_init, strict_invoke):
     assert not any(
         line.strip().lower().startswith("django") for line in result.output.splitlines()
     )
+
+
+@pytest.mark.integration
+@pytest.mark.parametrize("python_version", ["2.7", "3.6", "3.7", "3.8", "3.9"])
+def test_import_another_sitecustomize(python_version, project_no_init, strict_invoke):
```

Comment:
A smaller case around `pdm run` should be enough, isn't it? You can find some examples in `test_run.py`